sdk/python/pulumi_google_native/apigee/v1/host_security_report.py

```diff
@@ -324,7 +319,7 @@ def _internal_init(__self__,
             __props__.__dict__["self"] = None
             __props__.__dict__["state"] = None
             __props__.__dict__["updated"] = None
-        replace_on_changes = pulumi.ResourceOptions(replace_on_changes=["organization_id"])
+        replace_on_changes = pulumi.ResourceOptions(replace_on_changes=["organization_id", "*"])
```

Comment:
nit: If the merge includes a top-level wildcard, `*`, maybe we should drop other overrides?